### PR TITLE
add tests for the template namespace

### PR DESCRIPTION
**Pull request by jarofghosts**

### CODE REVIEW

File: src/doctopus/template.clj [hunk 1]
```diff
@@ -1,29 +1,29 @@
 (ns doctopus.template
   (:require [doctopus.doctopus :refer [list-heads list-tentacles]]
-   [net.cgrand.enlive-html :as enlive :refer [deftemplate defsnippet]]
+            [net.cgrand.enlive-html :as enlive :refer [deftemplate defsnippet]]
             [doctopus.configuration :refer [server-config]]
             [doctopus.doctopus :refer [list-heads list-tentacles list-tentacles-by-head]]
             [doctopus.doctopus.tentacle :refer [get-html-entrypoint]]
             [ring.util.anti-forgery :as csrf]))
 
-(defn- make-anchor
```
Comment (Gastove) [inline src/doctopus/template.clj:9]: If you ever need/want to, you _can_ actually reference private symbols in a test namespace -- something like `(#'namespace/private-fn)`.

**Comment by Gastove** [comment]
Dig it :+1:

File: src/doctopus/template.clj (hunk 1 above)
Comment (jarofghosts) [inline src/doctopus/template.clj:9]: Awesome! I looked around but couldn't find a way to do it. Will gladly update

Comment (Gastove) [inline src/doctopus/template.clj:9]: _really hopes he got that syntax right_

Comment (jarofghosts) [inline src/doctopus/template.clj:9]: turns out: you did! updated in https://github.com/Gastove/doctopus/commit/b7d69aa539967cab4b9a3f84c9394a74abc1d2b5

**Comment by Gastove** [comment]
Rock-awesome. Merge at will, Mr. Keane.

**Pull request merged**